charts/signoz/values.yaml

```diff
@@ -412,8 +428,8 @@ queryService:
   # ref: http://kubernetes.io/docs/user-guide/compute-resources/
   resources:
     requests:
-      cpu: 200m
-      memory: 300Mi
+      cpu: 100m
+      memory: 100Mi
     limits:
```

Comment:
@prashant-shahi I would not prefer having limits on query-service as it might need resources when running higher workload